conda_environment.yml

```diff
@@ -19,28 +19,30 @@ dependencies:
     # - futures  # for python 2 compatibility
 
     # core dependencies
+    - click
+    - distributed>=1.18
     - fiona
-    - rasterio
     - gdal
-    - numpy
     - geojson
-    - jinja2
-    - ulmo>=0.8.3.2
-    - pyyaml
-    - matplotlib
-    - click
-    - werkzeug
     - peewee
     - pandas<=0.22.0
     - geopandas
-    - distributed>=1.18
+    - jinja2
+    - jupyter
+    - matplotlib
+    - numpy
     - param
-    - psutil
```

Comment:
We need to leave this dependance in?

Reply:
Umm I'm not entirely sure, I've never really messed with this dependency before. Thanks

Reply:
Oh, I just alphabetized the list of dependencies and psutil was just moved. Thanks